### DISCUSSION TITLE
Contribution chart in the mobile app

### DISCUSSION THREAD

**Discussion by akctba** [community/community#141262] [Mobile]
We'd like to see the contribution chart in the app.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.181.0 (143346435)
Device: iPhone 13 (iOS 17.6.1)
</details>

**Comment by RahulScripted** [comment]


**Reply by akctba** [reply]
> Hi there, If you're having trouble viewing the contribution chart in the GitHub app for iOS, here are a few troubleshooting steps you can try:
> 
> 
> 
> 1. Check for updates: Ensure you're using the latest version of the GitHub app. You mentioned you're on version 1.181.0, so check the App Store to see if any new updates are available.
> 
> 
> 
> 2. Contact GitHub Support: If none of these solutions works, there may be an issue with the app itself. You can reach out to GitHub support for further assistance.

Rahul giving empty advice just for angagement. It's probably an AI answer because it's helps nothing.
